git-webui/src/share/git-webui/webui/js/git-webui.js

```diff
@@ -1524,7 +1524,20 @@ webui.DiffView = function(sideBySide, hunkSelectionAllowed, parent, stashedCommi
 
     var self = this;
 
-    self.update = function(cmd, diffOpts, file, mode) {
+    self.update = function(cmd, diffOpts, file, mode, reRun) {
```

Comment:
Cleaner approach: write a function that does everything after setting the 4 options, and call that instead of calling update with reRun set to true?